lib/banchan/workers/thumbnailer.ex

```diff
@@ -88,14 +133,33 @@ defmodule Banchan.Workers.Thumbnailer do
     File.mkdir_p!(Path.dirname(tmp_src))
     Uploads.write_data!(src, tmp_src)
 
+    if opts["isVideo"] do
+      duration = FFprobe.duration(tmp_src)
+
+      output_src = Path.join([System.tmp_dir!(), src.key <> ".jpeg"])
+
+      command =
+        FFmpex.new_command()
+        |> add_global_option(option_y())
+        |> add_input_file(tmp_src)
+        |> add_output_file(output_src)
+        |> add_file_option(option_f("image2"))
+        |> add_file_option(option_filter("scale=128:128"))
+        |> add_file_option(option_ss(round(duration / 2)))
+        |> add_file_option(option_vframes(1))
+
+      {:ok, _} = execute(command)
+
+      File.copy(output_src, tmp_src)
```

Comment:
```suggestion
      File.copy!(output_src, tmp_src)
```

This'll make it so we crash (good) if something goes wrong with the file copy.